Standards/scs-0001-v1-sovereign-cloud-standards.md

```diff
@@ -135,8 +145,25 @@ In addition, the following OPTIONAL sections should be considered:
   as a "table of contents" of things to work on in that context.
 - A _Related Documents_ section which references related Standards
   or Decisions, both upstream and/or other SCS documents.
-- A _Conformance Tests_ section that contains hints on how to validate
-  conformance with this spec, ideally links to conformance test cases.
+
+#### Decision Record
+
+Each Decision Record document MUST have the following sections:
+
+- An _Abstract_ providing a brief introduction on the topic of the document.
+- A _Context_ section describing the issue relevant for motivating this Decision Record.
+- A section containing the actual decision that is introduced. The section should also include
+  reasoning for this decision. We RECOMMEND naming the section _Decision_.
+
+We also RECOMMEND the following sections:
+
+- A _Terminology_ section which shortly describes terms used in the document, including possible abbreviations.
```

Comment:
```suggestion
- A _Terminology_ section which briefly describes terms used in the document, including possible abbreviations.
```

Reply:
Changed in 6fb17a19fc38f70a0dda6c5cc9689ff28d0b31e4